src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -78,246 +88,185 @@ public ProcessCommonJSModules(Compiler compiler, boolean reportDependencies) {
 
   @Override
   public void process(Node externs, Node root) {
```

Comment:
I was a bit confused at first, just because the root node isn't the usual jsRoot (the parent of all input files). Can we add a Preconditions check that root must be a SCRIPT node?